packages/forms/src/components/RichTextInput/MenuBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { Editor } from "@tiptap/react";
+import ListBulletIcon from "@heroicons/react/20/solid/ListBulletIcon";
+import ArrowUturnLeftIcon from "@heroicons/react/20/solid/ArrowUturnLeftIcon";
+import ArrowUturnRightIcon from "@heroicons/react/20/solid/ArrowUturnRightIcon";
+
+import { richTextMessages } from "@gc-digital-talent/i18n";
+
+import MenuButton from "./MenuButton";
+import LinkDialog from "./LinkDialog";
+
+interface MenuBarProps {
+  editor: Editor | null;
+}
+
+const MenuBar = ({ editor }: MenuBarProps) => {
+  const intl = useIntl();
+
+  return (
+    <div
+      data-h2-background="base(background.darkest)"
+      data-h2-padding="base(x.25)"
+      data-h2-display="base(flex)"
+      data-h2-gap="base(x.25)"
+      data-h2-radius="base(rounded rounded 0 0)"
+      data-h2-justify-content="base(space-between)"
+    >
+      <div data-h2-display="base(flex)" data-h2-gap="base(x.25)">
+        <MenuButton
+          active={editor?.isActive("bulletList") ?? false}
+          onClick={() => editor?.chain().focus().toggleBulletList().run()}
+          disabled={!editor?.isEditable || !editor?.can().toggleBulletList()}
+          icon={ListBulletIcon}
+        >
+          {intl.formatMessage(richTextMessages.bulletList)}
+        </MenuButton>
+        <LinkDialog editor={editor} />
+      </div>
```

Comment:
Is it possible for the editable prop to make the Menu inactive? Currently you can click on buttons to make dialogs pop up it just won't update the field with the value, just throw console errors

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/7b1e5b14-b314-4857-a2c7-73625688d9c0)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/3e4959b3-6c44-4da1-a2c1-30e3b3976a0b)

Reply:
Oh, huh. I thought it was supposed to with `!editor?.isEditable`. I guess not 😅  Thanks for catching that. I'll need to take a look.

Reply:
WOW! I knew it was working at one point. Looks like I refactored it a bit and forgot to pass the `editable: boolean` field. That was why this wasn't working 🤦‍♀️ 

Fixed in [fix read only mode](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7881/commits/34a380466694cfe447b8d6534e0478ffb26d97c0)